src/components/ModalPayment/ModalCardDetails/ModalCardDetails.tsx

```diff
@@ -203,16 +224,39 @@ const ModalCardDetails = ({
           'modalPayment.modalCardDetails.disclaimer.gift_card',
           sellerName
         );
+      case ModalPaymentTypes.modalPages.light_up_chinatown:
+        return t('modalPayment.modalCardDetails.disclaimer.light_up_chinatown');
       default:
         break;
     }
   };
 
+  const setDetailsText = (
+    type: string | ModalPaymentTypes.modalPages,
+    amount: number
+  ) => {
+    if (sellerId === 'send-chinatown-love')
```

Comment:
What is this used for?

Reply:
theres different text depending on the tier of the luc purchase

Reply:
Oops I wasn't clear. I was asking about `sellerId ===  'send-chinatown-love'`  since I don't think it would be used? I'm not too sure about this.

Reply:
oh this was a change made earlier. This is for if a user wants to donate to SCL but doesnt have a particular seller in mind. This route is identical to most of our donation copy, so we force the flow to to be the same as donations. I'm not sure who made this change though, maybe @danthaman44  ?